### PR TITLE
Dynamic Fields to use in repeater section acf

### PR DESCRIPTION
**Pull request by hamzahabib2000**
I made fields dynamic.
so everyone can use this in the repeater ...

### CODE REVIEW

**Comment by Beee4life** [comment]
Hi Hamza, it looks good at first sight, but need some time to review/test, won't be before this weekend or next....

**Comment by hamzahabib2000** [comment]
> 
> 
> Hi Hamza, it looks good at first sight, but need some time to review/test, won't be before this weekend or next....

Okay, I will followup this weekend ...

File: fields/acf-city_selector-v5.php [hunk 1]
```diff
@@ -128,17 +129,33 @@ function render_field( $field ) {
...
 					</select>
 				</div>
 
 				<div class="dropdown-box cs-cities">
 					<?php if ( $field['show_labels'] == 1 ) { ?>
 						<span class="acf-input-header"><?php esc_html_e( 'Select a city', 'acf-city-selector' ); ?></span>
 					<?php } ?>
-					<label for="cityName" class="screen-reader-text"></label>
-					<select name="<?php echo $field['name']; ?>[cityName]" id="cityName" class="countrySelect">
+					<?php $cities = get_cities( $countrycode, $field['value']['stateCode'] ); ?>
```
Comment (Beee4life) [inline fields/acf-city_selector-v5.php]: Undefined variable: countrycode

Comment (hamzahabib2000) [reply fields/acf-city_selector-v5.php:149]:
```
if ( isset( $field[ 'value' ][ 'countryCode' ] ) ) {
$countrycode = $field[ 'value' ][ 'countryCode' ];
}
```
okay, replace it to ...
```
$countrycode = null;
if ( isset( $field[ 'value' ][ 'countryCode' ] ) ) {
$countrycode = $field[ 'value' ][ 'countryCode' ];
}
```

Comment (Beee4life) [reply fields/acf-city_selector-v5.php:149]: Why did you place PHP code in an element which is generated by javascript ?

Comment (hamzahabib2000) [reply fields/acf-city_selector-v5.php:149]: I'm not sure what you wanna ask ... the element is called through ajax and the ajax requests dynamic file ....

Comment (Beee4life) [reply fields/acf-city_selector-v5.php:149]: You added option values for states and cities, where they are loaded dynamically. See [here](https://github.com/Beee4life/acf-city-selector/blob/master/assets/js/city-selector.js#L98) for states and [here](https://github.com/Beee4life/acf-city-selector/blob/master/assets/js/city-selector.js#L124) for cities. So I don't understand (yet) why you code this in when it's generated ?

**Comment by Beee4life** [comment]
I released a new version so that has to be pulled in your branch for it to be merged (IF).

**Comment by Beee4life** [comment]
I just checked your changes a bit more. I am sorry it doesn't work.

You string replace output the following: 

    <select name="acf[field_5e4c0e87789ce][5e9500baf4670][field_5e4c0e9a789cf][stateCode]" id="acffield_5e4c0e87789ce5e9500baf4670field_5e4c0e9a789cfstateCode" class="countrySelect">

Take a look at the ID, that can't be correct.

I appreciate your effort but this is not gonna work, sorry.

**Comment by Beee4life** [comment]
Your idea did gave me an idea on how to tackle this issue, so maybe in the next version (if I can fix it ;) )

**Comment by hamzahabib2000** [comment]
Hello Dear,
It's good to see you, we can work together and resolve many issues ....
join me at skype: hamzahabib89

**Comment by hamzahabib2000** [comment]
I would also ask about this is this working fine now ?

**Comment by Beee4life** [comment]
> I would also ask about this is this working fine now ?

I don't quite understand what you mean with 'this' ? I released a [new version](https://github.com/Beee4life/acf-city-selector/releases/tag/0.9) about 10 hours ago... After that I spent a few hours looking at your code and trying to implement it working, but didn't achieve it yet...

I said maybe in the next version, if I can fix it... That was after I released 0.9, so repeaters/groups are not fixed yet...

Reg. Skype no, sorry... That's why we have Github :)

**Comment by Beee4life** [comment]
Your setup did help me in the right direction. I think I have finally managed to make it dynamic in repeaters (didn't test groups yet). So I hope I can release it, this weekend at the latest.

**Pull request closed without merging**